jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -172,6 +172,11 @@ pipeline {
                             echo "Skipping tests for ${component_check} as is not present in the provided build manifest."
                             componentList -= component_check
                         }
+                    // Skipping integTest for cross-cluster-replication component when running on deb and rpm distribution
```

Comment:
Spacing.

Reply:
Please also add `TODO` to this comment and link the corresponding issue.
Eventually we would need to fix this as this change is temporary.

Thanks.